lib/containers/engine.pm

```diff
@@ -130,13 +130,14 @@ when it exits or when the daemon exits
 sub run_container {
     my ($self, $image_name, %args) = @_;
     die 'image name or id is required' unless $image_name;
-    my $mode = $args{daemon} ? '-d' : '-i';
-    my $remote = $args{cmd} ? "$args{cmd}" : '';
-    my $name = $args{name} ? "--name $args{name}" : '';
-    my $keep_container = $args{keep_container} ? '' : '--rm';
-    my $params = sprintf qq(%s %s %s), $keep_container, $mode, $name;
-    my $cmd = sprintf qq(run %s %s %s), $params, $image_name, $remote;
-    record_info "cmd_info", "Container executes:\noptions $params $image_name $remote";
+    # Assemble command: $RUNTIME run [-d|-i] [--rm] [--name $NAME] $IMAGE [$CMD]
```

Comment:
This belongs to perlpod IMO

Reply:
IMHO not because this is only a help to understand the next steps and not something that needs to be documented on the higher level. I could add it there as well though if you want?